configure.ac

```diff
@@ -59,22 +64,21 @@ RR_CHECK_FEATURE_LIB(TFLITE, TensorFlow lite Installation,
 # add specific LIBS for TFLITE
 AC_SUBST([TFLITE_LIBS], ["$TFLITE_LIBS -pthread -ldl -lrt"])
 
-AM_COND_IF([HAVE_EDGETPU], [AM_COND_IF([HAVE_TFLITE], [],
- [AC_MSG_ERROR(The EdgeTPU backend needs TFLITE enabled as well. Please enable it with enable-tflite option)])], [])
-
 RR_CHECK_FEATURE_LIB(TENSORRT, TensorRT Installation,
                      nvinfer, createInferBuilder_INTERNAL, NvInfer.h, no)
 
+# ONNXRT backend
+if test x$enable_onnxrt_acl = xyes || test x$enable_onnxrt_openvino = xyes; then
```

Comment:
Does this actually perform the checks needed to validate ONNXRT proper installation?

Reply:
Similar to comment above above, since we define enable_onnxrt=yes (which is equivalent to --enable-onnxrt from the command line) we are making sure that RR_CHECK_FEATURE_LIB macro acts as if the feature is enabled aswell.